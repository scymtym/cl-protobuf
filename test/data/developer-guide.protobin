
“
devloper-guide.protodeveloper_guide"
Test1	
a ("
Test2	
b (	"*
Test3!
c (2.developer_guide.Test1"
Test4
d (B