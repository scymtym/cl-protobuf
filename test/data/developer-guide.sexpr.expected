(
(:message developer-guide::test1
  (:field developer-guide::a :int32             1 :required t))

(:message developer-guide::test2
  (:field developer-guide::b :string            2 :required t))

(:message developer-guide::test3
  (:field developer-guide::c developer-guide::test1 3 :required t))

(:message developer-guide::test4
  (:field developer-guide::d :int32             4 :repeated t :packed t))
)